src/compositor.rs

```diff
@@ -1024,7 +1048,7 @@ impl IOCompositor {
         // Every display list needs a pipeline, but we'd like to choose one that is unlikely
         // to conflict with our content pipelines, which start at (1, 1). (0, 0) is WebRender's
         // dummy pipeline, so we choose (0, 1).
-        let root_pipeline = WebRenderPipelineId(0, 1);
+        let root_pipeline = WebRenderPipelineId(u64::from(self.current_window) as u32, 1);
```

Comment:
It should be the same. And yes, we create an additional pipeline to avoid conflict. This pipeline can be viewed as window's pipeline and should replace with latest display list when possible.